test/integration.test.js

```diff
@@ -51,19 +50,21 @@ describe('cached install', function() {
 
   it('creates .bin links when installing from cache', function(done) {
     // This test needs to install a package that is not already in PATH
-    // mkdirp is a good candidate, let's verify it's not already there
+    // cowsay is a good candidate, let's verify it's not already there
     try {
-      var mkdirpPath = which.sync('mkdirp');
-      return done(new Error('mkdir is already installed in ' + mkdirpPath));
+      var cowsayPath = path.resolve(SANDBOX, 'node_modules', '.bin', 'cowsay');
+      if (fs.accessSync(cowsayPath) || fs.accessSync(cowsayPath + '.cmd')) {
+        return done(new Error('mkdir is already installed in ' + cowsayPath));
```

Comment:
oops, was only looking for `mkdirp`.. I guess that means it was an existing bug? ;-)

Fixed in 0bc905f7efcae7d77f2f87ded1f4da26998d9750, which I just pushed straight to master.